### PR TITLE
Re-adding PDA Slots

### PR DESCRIPTION
**Pull request by AndrewMontagne**
Does exactly what it says on the tin. 👌 

:cl: AndrewMontagne
add: Re-adds PDA slots.
fix: "Set equipment" verb now strips all items before equipping.
/:cl:

### CODE REVIEW

**Comment by AndrewMontagne** [comment]
[PDA slot](https://trello.com/c/fHD4VtrF/39-pda-slot)

**Comment by AndrewMontagne** [comment]
TODO: Make pockets and pda slots get empties when "Select equipment" is invoked.

File: code/__DEFINES/inventory.dm [hunk 1]
```diff
@@ -28,6 +28,7 @@
 #define SLOT_POCKET		2048 // this is to allow items with a w_class of WEIGHT_CLASS_NORMAL or WEIGHT_CLASS_BULKY to fit in pockets.
 #define SLOT_DENYPOCKET	4096 // this is to deny items with a w_class of WEIGHT_CLASS_SMALL or WEIGHT_CLASS_TINY to fit in pockets.
 #define SLOT_NECK		8192
+#define SLOT_PDA		16384 // Do not remove: Causes the community to get angry
```
Comment (SamHPurp) [inline code/__DEFINES/inventory.dm]: 👌

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
 	flags_1 = NOBLUDGEON_1
 	w_class = WEIGHT_CLASS_TINY
-	slot_flags = SLOT_ID | SLOT_BELT
+	slot_flags = SLOT_ID | SLOT_BELT | SLOT_PDA
```
Comment (SamHPurp) [inline code/game/objects/items/devices/PDA/PDA.dm]: Do we want to remove the `SLOT_ID` from this as well. Pretty sure its that way elsewhere? (I could be entirely wrong)

Comment (AndrewMontagne) [reply code/game/objects/items/devices/PDA/PDA.dm]: This is how it is on Paradise.

**Comment by AndrewMontagne** [comment]
@SamHPurp The IDs spawn in the ID slot.

**Comment by FlattestGuitar** [comment]
👌

**Pull request merged**